appstore/discord.py

```diff
@@ -0,0 +1,112 @@
+import json
+import requests
+import random
+
+from .settings import config
+from .utils import get_app_description
+from appstore.models import App
+
+party_time_emoji = ["🎉","🥳","👏","❤️","🥰","🎊"]
+
+def random_party_emoji():
+    return random.choice(party_time_emoji)
+
+def announce_release(App, Release):
+    try:
+
+        release_notes = Release.release_notes
+        if len(release_notes) < 1:
+            release_notes = "N/A"
+
+        request_data = {
+            "embeds": [{
+                "title": f"{str(App.type).capitalize()} Update Alert {random_party_emoji()}",
+                "url": f"{config['APPSTORE_ROOT']}/en_US/application/{App.id}",
+                "thumbnail": {
+                    "url": f"{config['IMAGE_ROOT']}/80x80/filters:upscale()/{App.icon_large}",
+                    "height": 80,
+                    "width": 80
+                },
+                "description": f"{App.developer.name} just updated their {App.type} *{App.title}* to version {Release.version}!",
+                "fields": [
+                    {
+                        "name": "Release Notes",
+                        "value": release_notes
+                    }
+                ]
+            }]
+        }
+
+        send_discord_webhook(request_data)
+
+    except Exception as e:
+        # Let's not make a fuss
+        return
```

Comment:
This decision should be made by the caller; exceptions should propagate and/or be transformed from here.